web-app/uni-todo-app/src/components/TodoListItem.tsx

```diff
@@ -15,12 +15,12 @@ const TodoListItem = ({
     <>
       <div className="flex mb-4 items-center">
         {!todoItem.done && (
-          <p className="w-full text-grey-darkest">{todoItem.name}</p>
+          <p className="w-full text-grey-darkest">{todoItem.name}<p>Created on {new Date(todoItem.createdAt).toLocaleDateString()}</p></p>
         )}
 
         {todoItem.done && (
           <p className="w-full line-through text-grey-darkest">
-            {todoItem.name}
+            {todoItem.name}<p>Created on {new Date(todoItem.createdAt).toLocaleDateString()}</p>
```

Comment:
To minimize duplicate code and make it more understandable, you could create a function that gets the date as param and returns the formated string. It also makes the code cleaner and easier to understand :)